source/website/src/views/Step3.vue

```diff
@@ -380,11 +381,12 @@ SPDX-License-Identifier: Apache-2.0
             this.isBusy = true;
             this.column_type_options.forEach(x => x.disabled = false)
             this.pii_type_options.forEach(x => x.disabled = false)
+            importJson = this.resolveKeys(importJson)
             this.items = importJson.columns.map(x => {return {
                 "name": x.name,
                 "description": x.description.charAt(0).toUpperCase() + x.description.replace(/[^a-zA-Z0-9]/g, ' ').slice(1),
-                "data_type": x.data_type,
-                "column_type": x.column_type,
+                "data_type": x.data_type || x.dataType,
```

Comment:
We shouldn't be trying to support two naming conventions. If you want to use a different convention, then rename the old vars.

Reply:
ok i will remove this.

Reply:
Resolved